examples/chip-tool/commands/discover/DiscoverCommand.h

```diff
@@ -36,6 +36,7 @@ class DiscoverCommand : public Command, public chip::Controller::DeviceAddressUp
 
     /////////// Command Interface /////////
     CHIP_ERROR Run() override;
+    uint16_t GetWaitDurationInSeconds() const override { return 30; }
```

Comment:
LGTM, Just wondering about the different `GetWaitDurationInSeconds` values here and in following  command interfaces. How were they determined?

Reply:
@bzbarsky-apple can you follow up here?

Reply:
I kept the existing values, which I suspect were determined by "this should not take longer than, oh I guess this long" or so.